src/docs/common/snippets/common-mysql-datasource-without-test-resources.adoc

```diff
@@ -0,0 +1,22 @@
+== Connecting to a MySQL database
+
+Previously, we connected to a MySQL database, which Micronaut Test Resources started for us.
+
+However, it is easy to connect to an already existing database. Let's start a database and connect to it.
+
+common:run-mysql-container.adoc[]
+
+Export several environment variables:
+
+[source,bash]
+----
+export DATASOURCES_DEFAULT_URL=jdbc:mysql://localhost:3306/db
+export DATASOURCES_DEFAULT_USERNAME=sherlock
+export DATASOURCES_DEFAULT_PASSWORD=elementary
+----
+
+Micronaut Framework populates the properties `datasources.default.url`, `datasources.default.username` and `datasources.default.password` with those environment variables' values.
+
+You can run the application and test the API as it was described in the previous sections. However, when you run the application, Micronaut Test Resources does not start a MySQL container because you have provided values for `datasources.default.*` properties.
+
```

Comment:
Might want to link to documentation, for example: https://micronaut-projects.github.io/micronaut-sql/latest/guide/#jdbc-connection-pools.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-guides/pull/1089/commits/990f0a5c0688dfbe740dbb5f0f0a32d16c156d2f